### DISCUSSION TITLE
which model is siamrpn++?

### DISCUSSION THREAD

**Discussion by codylcs** [STVIR/pysot#447] [Q&A] [answered]
I am confused about the names of different models,which one is SiamRPN,and which one is SiamRPN++？
![image](https://user-images.githubusercontent.com/48380363/95941536-92809580-0e13-11eb-848a-54f367bcc66d.png)

**Comment by ZhiyuanChen** [comment, marked as answer]
All of them other than `siammask_r50_l3`

Update:
`siamrpn_r50_l234_dwxcorr`, `siamrpn_r50_l234_dwxcorr_otb` and `siamrpn_r50_l234_dwxcorr_lt` could be the ones that are "most" siamrpn++
